.drone.yml

```diff
@@ -1,30 +1,96 @@
 workspace:
   base: /go
+  path: src/github.com/drone-plugins/drone-slack-blame
 
 pipeline:
   test:
-    image: golang:1.6
-    environment:
-      - CGO_ENABLED=0
+    image: golang:1.9
     commands:
       - go vet
       - go test -cover -coverprofile=coverage.out
-      - go build -ldflags "-s -w -X main.build=$DRONE_BUILD_NUMBER" -a -tags netgo
 
-  latest:
+  build_linux_amd64:
+    image: golang:1.9
+    group: build
+    environment:
+      - GOOS=linux
+      - GOARCH=amd64
+      - CGO_ENABLED=0
+    commands:
+      - go build -v -ldflags "-X main.build=${DRONE_BUILD_NUMBER}" -a -tags netgo -o release/linux/amd64/drone-slack-blame
```

Comment:
You don't need `-tags netgo` to build static binaries anymore since Go 1.5.

Reply:
This is a plain copy from the .drone.sh script provided by @bradrydzewski, but from my side I'm fine to update it like that